format/spec.md

```diff
@@ -193,10 +193,38 @@ Notes:
+| **`string`**       | **`string`**  | "foo"                                  |                                                                                                                                                                                                                                              |
+| **`uuid`**         | **`string`**  | "eb26bdb1-a1d8-4aa6-990e-da940875492c" |                                                                                                                                                                                                                                              |
+| **`fixed(L)`**     | **`string`**  | "0x3162"                               | we use hexadecimal byte literals to encode bytes, with prefix `0x`                                                                                                                                                                           |
+| **`binary`**       | **`string`**  | "0x3162"                               | we use hexadecimal byte literals to encode bytes, with prefix `0x`                                                                                                                                                                           |
+| **`struct`**       | **`object`**  | {"a": 1, "foo": "bar"}                 | the keys are the nested fields' names in the struct schema, and the values should be corresponding value literals of the field's type                                                                                                        |
+| **`list`**         | **`array`**   | \[1, 2, 3\]                            | each value should be value literals of the corresponding element type of the list                                                                                                                                                            |
+| **`map`**          | **`object`**  | {"a": 1, "b": 2}                       | the key and value type correspond with the schema of the map, the key (json string) are parsed into the resulting key type of the map schema, an exception will be thrown if parsing fails.                                                  |
+
+Default value is treated as an attribute of a nested field in the table schema when the schema is serialized, see JSON encoding in [Appendix C](#Appendix-C:-JSON-serialization)).
 
 #### Schema Evolution
 
-Schemas may be evolved by type promotion or adding, deleting, renaming, or reordering fields in structs (both nested structs and the top-level schema’s struct).
+Schemas may be evolved by type promotion or adding, deleting, renaming, or reordering fields in structs (both nested structs and the top-level schema’s struct). Setting a default value for a field is also considered a schema evolution.
```

Comment:
I don't think that this is needed.

Setting a default value can only be done when adding a field, which is already an evolution.